tests/src/rules/group-exports.js

```diff
@@ -0,0 +1,70 @@
+import { test } from '../utils'
+import { RuleTester } from 'eslint'
+import rule from 'rules/group-exports'
+
+const ruleTester = new RuleTester()
+
+ruleTester.run('group-exports', rule, {
+  valid: [
+    test({ code: 'export const test = true' }),
+    test({ code: 'export default {}\nexport const test = true' }),
+    test({ code: [
+      'const first = true',
+      'const second = true',
+      'export { first,\nsecond }',
+    ].join('\n') }),
+    test({ code: 'export default {}\n/* test */\nexport const test = true'}),
+    test({ code: 'export default {}\n// test\nexport const test = true'}),
+    test({ code: 'export const test = true\n/* test */\nexport default {}'}),
+    test({ code: 'export const test = true\n// test\nexport default {}'}),
+    test({ code: 'module.exports = {} '}),
+    test({ code: 'module.exports = { test: true,\nanother: false }' }),
+    test({ code: 'exports.test = true' }),
+  ],
+  invalid: [
+    test({
+      code: [
+        'export const test = true',
+        'export const another = true',
+      ].join('\n'),
+      errors: [
+        'Multiple named export declarations',
```

Comment:
Messages updated as requested.

Autofixing would be quite difficult, it seems - we have to
- remove `export` from in front of the things being exported
- add one extra `export` declaration somewhere into the file with all the exported names

The first step seems easy, but I am unsure how to do the second, given you can do only one fix per rule violation.

I guess we could use the last found export and do a full replacement of the source where we append the export at the end...Not sure if this is a good idea, though - if the code we replace contains other issues, I have no idea how ESLint would solve that.